.travis.yml

```diff
@@ -3,6 +3,8 @@ dist: xenial
 services:
   - xvfb
 
+cache: pip
```

Comment:
if im not wrong, this means `~/.cache/pip`

Reply:
Yes, `$HOME/.cache/pip`, according to [the documentation](https://docs.travis-ci.com/user/caching/#pip-cache)